## Summary

This function throws the exception that it is given as an argument.

## Signature

raise (exception : [Throwable](https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html)) : void

## Formals

+ Parameter <i>exception</i> is the exception to throw.

## Returns

This function does not return, because the <i>exception</i> is thrown.

## Throws

+ [Throwable](https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html), in order to be capable of throwing any type of exception.
+ [NullPointerException](https://docs.oracle.com/javase/7/docs/api/java/lang/NullPointerException.html), if <i>exception</i> is null.

## Example

**Source Code:**

```plain
module Main in execution;

@Start
defun main (args : String[]) : void
{
    val t = new RuntimeException(); 

    try
    {
        F::raise(t);

        F::println("The exception was not thrown.");
    }
    catch(ex : RuntimeException)
    {
        F::println("The exception was thrown.");
    }
}
```

**Output:**

```plain
The exception was thrown.
```

